charts/hail-search/values.yaml

```diff
@@ -30,40 +30,18 @@ resources:
 
 # default environment variables
 environment:
-  DATASETS_DIR: '/datasets'
-  SSD_DATASETS_DIR: '/ssd-datasets'
+  HAIL_SEARCH_DATA_DIR: '/seqr/seqr-hail-search-data'
+  REFERENCE_DATASETS_DIR: '/seqr/seqr-reference-data'
+  IN_MEMORY_DIR: '/in-memory-datasets'
 
+inMemoryVolumeEnabled: false
```

Comment:
Not a blocker, but it feels a bit weird to configure `IN_MEMORY_DIR` by default but also set  `inMemoryVolumeEnabled` as false by default. I'm not actually convinced it makes sense to include the configurations for in-memory in the open source chart at all

Reply:
yeah I agree with this and had the same thought.  I defined `IN_MEMORY_DIR` because it'd throw an exception if `inMemoryVolumeEnabled` was enabled.  It's very possible (though annoying) to make `volumes` and `volumeMounts` lists with no reference to in memory anything.  I can do that.

Reply:
I did this!  It wasn't terrible.  Now the `in-memory` stuff lives out of the core chart and is only in the unit tests/`tgg-helm`, which is better.